static/app/views/organizationStats/mapSeriesToChart.ts

```diff
@@ -158,6 +154,10 @@ export function mapSeriesToChart({
       });
     });
 
+    // Combine rate limited counts
+    count[Outcome.RATE_LIMITED] +=
+      count[Outcome.ABUSE] + count[Outcome.CARDINALITY_LIMITED];
+
```

Comment:
This final summing-up pf the count was inside the loop that calculated the counts, causing a far too high value for `Outcome.RATE_LIMITED`.